tests/integration/test.basics.js

```diff
@@ -73,6 +73,20 @@ adapters.forEach(function (adapter) {
       return new PouchDB(dbs.name).destroy({});
     });
 
+    it('4339 throw useful error if method called on stale instance', function () {
+      var db = new PouchDB(dbs.name);
+
+      return db.put({
+        _id: 'cleanTest'
+      }).then(function () {
+        return db.destroy();
+      }).then(function () {
+        return db.get('cleanTest');
+      }).catch(function (err) {
+        should.equal(err instanceof Error, true, 'should be an error');
+      });
```

Comment:
Sorry I missed this point because I was thinking of the catch behaviour when it's chained so I was confused why you'd be 'forcing' the catch the be called.

Infact here the error thrown if the get passes will not get caught as it's not chained, thanks for the tip.